specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2020-10-01/StorageInsightConfigs.json

```diff
@@ -0,0 +1,335 @@
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "StorageInsights"
+        ],
+        "x-ms-examples": {
+          "StorageInsightsDelete": {
+            "$ref": "./examples/StorageInsightsDelete.json"
+          }
+        },
+        "operationId": "StorageInsightConfigs_Delete",
+        "description": "Deletes a storageInsightsConfigs resource",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "name": "storageInsightName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of the storageInsightsConfigs resource"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition."
+          },
+          "204": {
+            "description": "NoContent response definition."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/storageInsightConfigs": {
+      "get": {
+        "tags": [
+          "StorageInsights"
+        ],
+        "x-ms-examples": {
+          "StorageInsightsList": {
+            "$ref": "./examples/StorageInsightsListByWorkspace.json"
+          }
+        },
+        "operationId": "StorageInsightConfigs_ListByWorkspace",
+        "description": "Lists the storage insight instances within a workspace",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition.",
+            "schema": {
+              "$ref": "#/definitions/StorageInsightListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "@odata.nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "StorageAccount": {
```

Comment:
> "StorageAccount": { [](start = 3, length = 20)

It looks like the difference between the last version of StorageInsightConfigs(2020-08-01) and this one is the model of the StorageAccount - is that right?